pkg/ibmvpc/examples/out/analysis_out/tgw_basic_example_dup_subnet_names_all_vpcs_subnetsBased_withPGW_with_grouping.txt

```diff
@@ -0,0 +1,9 @@
+Connectivity between VPCs connected by TGW local-tg-ky (UID: crn:161)
+ky-vpc1-net1 => ky-vpc1-net1 : All Connections
+ky-vpc1-net1 => ky-vpc1-net1 : All Connections
```

Comment:
open an issue with reference to this example output, where subnet name should also include vpc , at least in the cross-vpc connectivity ?

Reply:
Sure.  https://github.com/np-guard/vpc-network-config-analyzer/issues/498
But note that we can also have two subnets with the same name in the same VPC (though one may claim in is less likely); As I have used this input file to demonstrate the dup names issue I could have chosen another one with only one VPC.
If we really want to tackle dup names we should perhaps issue some lint error. Or have some renaming mechanism? perhaps open issue for this as well?

Reply:
ok, updated the issue to contain these as two related tasks